witchcraft-server/src/service/request_log.rs

```diff
@@ -119,25 +120,25 @@ where
         };
 
         let mdc = mdc::snapshot();
-        let uid = mdc
```

Comment:
We don't want the org ID in service logs for now so it can't go in the MDC. I just switched everything over to directly loading the UnverifiedJwt extension as a result.